airbyte-integrations/bases/base-normalization/normalization/transform_catalog/destination_name_transformer.py

```diff
@@ -74,6 +76,8 @@ def needs_quotes(self, input_name: str) -> bool:
             return True
         if self.destination_type.value == DestinationType.BIGQUERY.value:
             return False
+        if self.destination_type.value == DestinationType.ORACLE.value and input_name.startswith("_"):
+            return True
```

Comment:
With ORACLE, when a column name starts with `_`, do you quote it? or do you remove the '_' character? or both?

Reply:
you have the option to quote the column but the airbyte default columns we need to remove `_` because is what the destination connector is doing...